### PR TITLE
[Changelog] Update for Sprint 88

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
+## Unreleased as of Sprint 88 ending 2018-06-18
+
+### Added
+- Expose task.mark_vm_migrated to Service. [(#186)](https://github.com/ManageIQ/manageiq-automation_engine/pull/186)
```
Comment (gmcculloug) [inline CHANGELOG.md]: Expose `mark_vm_migrated` method to TransformationPlanTask service model.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
...
+
+### Fixed
+- Adding the `storages` attribute to be exposed. [(#187)](https://github.com/ManageIQ/manageiq-automation_engine/pull/187)
```
Comment (gmcculloug) [inline CHANGELOG.md]: Expose `storages` to VmOrTemplate service model.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
...
+- Service model: method exposing [(#175)](https://github.com/ManageIQ/manageiq-automation_engine/pull/175)
```
Comment (gmcculloug) [inline CHANGELOG.md]: Exclude private model methods from being exposed in the Service model.

**Pull request merged**